botocore/loaders.py

```diff
@@ -353,7 +389,7 @@ def load_service_model(self, service_name, type_name, api_version=None):
         return self.load_data(full_path)
 
     @instance_cache
-    def load_data(self, name):
+    def load_data(self, name, use_ordered_dict=True):
```

Comment:
Hmm I wondering if there is a better way to expose ``use_ordered_dict`` value as this will only be applicable to JSON loading. So if we want to add a plain text loader as the ``FILE_LOADER_CLASS`` this would not make much sense. I will try to think of something.